src/services/info/sv-compass-feature.ts

```diff
@@ -0,0 +1,44 @@
+import { Feature } from 'ol'
+import { FeatureLike } from 'ol/Feature'
+import { Style, Icon } from 'ol/style'
+import directionUrls from '@/services/info/sv-direction-urls'
+
+export class SvCompassFeature extends Feature {
+  zoom: number
+  heading: number
+  pitch: number
+
+  getStyleFunction() {
+    return function (feature: FeatureLike /*, resolution: number*/): Style[] {
```

Comment:
why do you keep comments like that?

Reply:
because basically it is a 2 argument callback function but the linter complains about the second argument not being used, so this keeps them visible